benchexec/tablegenerator/__init__.py

```diff
@@ -1382,10 +1386,201 @@ def write_head_line(
         out.write("\n")
 
 
+def write_tex_command_table(
+    out,
+    run_sets: List[RunSetResult],
+    stats: List[List[ColumnStatistics]],
+    **kwargs,
+):
+    bench_name_set = set()
```

Comment:
The abbreviation `bench_name` is so far never used in BenchExec. Can you please use a more common term like `benchmark_name`? (Also in all other occurrences of `bench_name`, please.)